lib/inputs/binaries/static/index.ts

```diff
@@ -0,0 +1,24 @@
+import { ExtractAction, ExtractedLayers } from "../../../extractor/types";
+import { streamToHash } from "../../../stream-utils";
+
+export const getNodeBinariesFileContentAction: ExtractAction = {
+  actionName: "node",
+  fileNamePattern: "**/node",
+  callback: streamToHash,
+};
+
+export function getBinariesHashes(extractedLayers: ExtractedLayers): string[] {
+  const hashes: Set<string> = new Set<string>();
+  for (const fileName of Object.keys(extractedLayers)) {
+    for (const actionName of Object.keys(extractedLayers[fileName])) {
+      if (actionName !== getNodeBinariesFileContentAction.actionName) {
+        continue;
+      }
+      if (!(typeof extractedLayers[fileName][actionName] === "string")) {
+        throw new Error("expected string");
+      }
+      hashes.add(extractedLayers[fileName][actionName] as string);
```

Comment:
will this solve the problem of files in the same locations? looks like it will just dedup in cases the hash repeats twice (and not if the path returns twice)

Reply:
not sure what this means, please read my response on the previous comment and let me know if something doesn't make sense https://github.com/snyk/snyk-docker-plugin/pull/139#discussion_r383237088